docs/src/dev/provider/index.asciidoc

```diff
@@ -1033,6 +1036,21 @@ WebSocket, Gremlin Server can accept a packet frame using a "text" (1) or a "bin
 is a bit more limited in that Gremlin Server will always process the body of that request as JSON.  Generally speaking
 "text" is just for testing purposes.
 
+For HTTP can be used following serializers distinguished by `Content-Type` HTTP header
+[width="100%",cols="3,10, 10",options="header"]
+|=========================================================
+|Serializer | Content-Type| Description
+|GraphBinaryMessageSerializerV1 | application/vnd.graphbinary-v1.0 | Efficient binary format recommended for default use.
+|GraphSONMessageSerializerV3 | application/vnd.gremlin-v3.0+json | Json-compatible format containing types of all data.
+|GraphSONUntypedMessageSerializerV3 | application/vnd.gremlin-v3.0+json;types=false | Json-compatible format without metadata. For this reason it cannot be used for transmission of ByteCode requests.
+|=========================================================
```

Comment:
I wonder if we want to give a bit more details on the design? E.g. how the request is handled when it comes in? Why we are using Content-Type, what should the payload looks like, and some notes on what happens if you used different serializers for request/response? Also some potential issue people may encounter with this (especially if spec isn't followed)?

Also I wonder if this needs to be in a separate section, it feels a bit odd being sticked into the middle of this doc, but I don't have any good recommendations right now either.

Reply:
I'd echo @xiazcy concerns here. I sense that it's possible that some of the docs have even fallen behind here as I read the content around what's been added. This feature remains experimental so when I was thinking about documentation here I was mostly thinking about more extensive javadoc rather than official changes to asciidoc. When the experimental label is dropped we can add this sort of content.